content/current/doap_Gora.rdf

```diff
@@ -36,7 +36,7 @@
     <mailing-list rdf:resource="http://gora.apache.org/mailing_lists.html" />
     <download-page rdf:resource="http://gora.apache.org/releases.html" />
     <programming-language>Java</programming-language>
-    <category rdf:resource="http://projects.apache.org/category/nosql" />
+    <category rdf:resource="http://projects.apache.org/category/retired" />
```

Comment:
Just add new category instead of replace existing one:

```
    <category rdf:resource="http://projects.apache.org/category/nosql" />
    <category rdf:resource="http://projects.apache.org/category/retired" />
```

Reply:
yes, project will have 2 categories: keeping the nosql category remains valuable

Reply:
notice I'll update our documentation https://attic.apache.org/process-howto.html#4-update-the-project-doap-file-if-any to help clarifying that it;s about addiing, not replacing